src/plot/plot.jl

```diff
@@ -48,26 +53,75 @@ function clusters(
 
     clusters!(
         ax,
-        clusters=clusters,
-        cluster_sequence=cluster_sequence,
-        cluster_radius=cluster_radius,
-        nodes=nodes,
-        centers=centers,
-        labels=labels
+        clusters,
+        cluster_radius,
+        nodes,
+        centers,
+        labels
     )
 
     return fig, ax
 end
+function clusters(
+    ;
+    cluster_sequence::Union{DataFrame,Nothing}=nothing,
+    cluster_radius::Real=0,
+    centers::Bool=false,
+    labels::Bool=false
+)::Tuple{Figure,Axis}
+    fig = Figure(size=(800, 600))
+    ax = Axis(fig[1, 1], xlabel="Longitude", ylabel="Latitude")
+
+    clusters!(
+        ax,
+        cluster_sequence;
```

Comment:
Flagging that this feels weird - I think better to dispatch on whether you pass in a cluster sequence or not?